CONTRIBUTING.md

```diff
@@ -35,3 +35,10 @@ Must be one of the following:
 - **refactor**: A code change that neither fixes a bug nor adds a feature
 - **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
 - **test**: Adding missing tests or correcting existing tests
+
+
+### Becoming an EddieHub Maintainer
+
+Eddiehub is all about open source collaboration. We want it to be a place which is inclusive and welcoming for all and therefore our maintainers are carefully chosen from those we see upholding the Eddiehub ethos of collaboration first, code second, making quality contributions to Eddiehub over a longer period of time and those that follow the [CODE of CONDUCT](CODE_OF_CONDUCT.md). 
```

Comment:
```suggestion
EddieHub is all about open source collaboration. We want it to be a place which is inclusive and welcoming for all and therefore our maintainers are carefully chosen from those we see upholding the EddieHub ethos of collaboration first, code second, making quality contributions to EddieHub over a longer period of time and those that follow the [CODE of CONDUCT](CODE_OF_CONDUCT.md). 
```

Reply:
what is ethos?

Reply:
Ethos means guiding principles. I should maybe change it to make it easier to understand. Anything else which seems unclear?

Reply:
> Ethos means guiding principles. I should maybe change it to make it easier to understand. Anything else which seems unclear?

Ok cool, Nope nothing else is unclear.